staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -82,7 +81,6 @@ func New(
 	serverConcurrencyLimit int,
 	requestWaitLimit time.Duration,
 ) Interface {
-	grc := counter.NoOp{}
```

Comment:
IIUC counter.NoOp is no longer used anywhere - so let's just remove it.

Reply:
done